### PR TITLE
Smart Futures blog post

### PR DESCRIPTION
**Pull request by dmateusp**
Smart Futures is an Ireland based initiative to get children excited about S.T.E.M subjects.

At the Dublin office we started getting involved and we registered ~10 volunteers; this is a post talking about volunteering and specifically about Smart Futures

We gave 2 presentations in a school a few months back, so it also includes some tips

Objective is to spread the word, both internally and among other companies in Ireland. But also hopefully inspire potential volunteers in other Tech companies to join their local initiatives.

### CODE REVIEW

File: _posts/2018-04-16-smart-futures.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- culture
+#
+tags:
+- stem
+- initiatives
+- children
+- volunteering
+---
+
+With so many great tech initiatives out there and limited time to contribute, it can be a though choice for a volunteer to pick one (or a few) initiatives to get involved with.
```
Comment (krajasethupathy) [inline _posts/2018-04-16-smart-futures.md]: though -> tough

**Comment by krajasethupathy** [comment]
very cool @dmateusp, and i love that approach of motivating them to ask questions as it encourages them to approach something from their perspective and form their own models for how things work. is it kosher to add video footage of an interaction between you guys and the kids? might be instructive to blog readers...

**Comment by KarlSimon100** [comment]
+1 on @krajasethupathy comments, @dmateusp . It's really great that you're doing this, and, the blog post has good structure in terms of included sections, and content within.

Additionally, I have the following recommendations...
----------------------------------------------------------------------------------------
The sections/content appears a bit jumbled (_including a ‘header-free’ excerpt at the top, appearing to most appropriate complete the ‘How to pick an initiative’ section_).
*What is Smart Futures*
*Why it’s awesome*
*How to pick an initiative*
*Our experience*
*How do I get involved*
*Help us find schools*

In the *Why it’s awesome* section, I see the great intent to make a powerful, opening statement to highlight the volunteer’s involvement and make a big difference helping a student choose his/her path.  At the moment though, the opening statement has the risk of unintended misinterpretation, w.r.t. clarifying the obligation that someone must physically come to the schools.  I would consider rewriting as follows:

“The world needs more S.T.E.M.-based (or minimally, “aware”) students, and, *you* are acting as a potential change agent in many students’ education path selection.  Those very same students attending your presentation may additionally influence others not in attendance.

Consider your role in changing people’s understanding and perspective:
- These children have never signed up to to a coding class - they come from all backgrounds, and might not have any relatives or friends in the industry;
- By going to their schools, your provide them with an opportunity to concretely picture ‘Tech’, generate excitement about potential projects, and obtain answers about the inner workings of applications they use every day (at a high level)!

To quote the website: …”

In the *Our experience* section, I would extend the first sentence to provide your retrospective on the results of your approach with Jake Whelan, indicating some level of success and/or lessons learned, if applicable.

“Jake Whelan and I have given 2 presentations to classes of ~50 children aged 10-11.  We found the following approach increased the level of engagement and participation throughout, with some exceptions (e.g. x, y, z).”

File: _posts/2018-04-16-smart-futures.md (hunk 1 above)
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: oops!

File: _posts/2018-04-16-smart-futures.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Different initiatives target different issues (e.g. diversity, education) and as a volunteer your tasks will vary (e.g. teach children, present to adults, organise).
```
Comment (tmccartan) [inline _posts/2018-04-16-smart-futures.md]: organise what? should this be something like ad-hoc organisation tasks?

File: _posts/2018-04-16-smart-futures.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+---
+
+With so many great tech initiatives out there and limited time to contribute, it can be a though choice for a volunteer to pick one (or a few) initiatives to get involved with.
+
+Different initiatives target different issues (e.g. diversity, education) and as a volunteer your tasks will vary (e.g. teach children, present to adults, organise). 
+
+# How to pick an initiative 
+
+There are 4 questions I would ask myself as a volunteer looking to get involved; I expect the answers to be very subjective, but I included my answers for reference:
+  1. what issues matter the most to me ?
```
Comment (tmccartan) [inline _posts/2018-04-16-smart-futures.md]: Use Captials here? What, How,Do, Is

File: _posts/2018-04-16-smart-futures.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  
+  3. do I have other expectations about my volunteering experience ?
+  
+        I like teaching, explaining to children and presenting in general
+        
+  4. is there an existing initiative aligning with my goal ?
+        
+        Yes at least 2 that I know of: [CodeDojo](https://coderdojo.com/) and [Smart Futures](http://www.smartfutures.ie/)
+        
+Depending on where you live and where you work, you will have more or less options; but keep in mind that some initiatives like [CodeDojo](https://coderdojo.com/) will help you set up local Computing classes and have a lot of resources that can help you.
```
Comment (tmccartan) [inline _posts/2018-04-16-smart-futures.md]: Code Dojo or Coder Dojo?

File: _posts/2018-04-16-smart-futures.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+>Many students have no access to anyone working in science and technology, or may not know anyone that went to college.
+
+# Our experience
+
+[Jake Whelan](https://ie.linkedin.com/in/jakewhelan) and I have given 2 presentations to classes of ~50 children aged 10-11.
+
+We figured that children of that age want to have examples as concrete as possible, and decided not to talk about our company too much as eCommerce might be abstract for children of that age (probably non-shoppers!).
+
+Our presentation kept the introductions short, we showed a very simple JavaScript-based game that we created (along with the code) and made sure to keep the presentation as interactive as possible.
```
Comment (tmccartan) [inline _posts/2018-04-16-smart-futures.md]: Is the code for this opensourced? Might be worth adding in a link

File: _posts/2018-04-16-smart-futures.md (hunk 2 above)
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: oh good point, I'll add some details.. I was thinking about organisation of events

File: _posts/2018-04-16-smart-futures.md (hunk 3 above)
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: will probably look better indeed :p

File: _posts/2018-04-16-smart-futures.md (hunk 4 above)
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: typo!

File: _posts/2018-04-16-smart-futures.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+With so many great tech initiatives out there and limited time to contribute, it can be a though choice for a volunteer to pick one (or a few) initiatives to get involved with.
+
+Different initiatives target different issues (e.g. diversity, education) and as a volunteer your tasks will vary (e.g. teach children, present to adults, organise). 
+
+# How to pick an initiative 
+
+There are 4 questions I would ask myself as a volunteer looking to get involved; I expect the answers to be very subjective, but I included my answers for reference:
+  1. what issues matter the most to me ?
+        
+        students choosing CS (Computer Science) in college do not have a concrete idea of what CS is, I found that many class-mates were deceived as they had a false and stereotypical expectation of what CS would be; on the other side: many highschool class-mates never even considered CS either (for the same reason) and might have missed a passion!
```
[the remark this answers is not in the document]
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: it does look weird now that I re-read it

File: _posts/2018-04-16-smart-futures.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+categories:
+- culture
+#
+tags:
+- stem
+- initiatives
+- children
+- volunteering
+---
```
[the remark this answers is not in the document]
Comment (dmateusp) [reply _posts/2018-04-16-smart-futures.md]: not sure what pictures we can use from Smart futures, @amccabe-gilt do you know if they sent us some pictures for presentations ?

**Comment by dmateusp** [comment]
@KarlSimon100 thank you for taking the time to write this really detailed feedback ^^

1. the decision for header-free is that the actual title is taken from the meta-data in the top of the blog markdown see http://tech.hbc.com/2018-03-23-negative-sampling-in-numpy.html as an example

2. what you're saying about the order makes sense!

3. I'll take that in account and try to make the "Why it's awesome" part more inspiring

4. You make a good point there, I need to provide more details on Our experience: @tmccartan asked about code resources that we can provide, and @krajasethupathy about video footage (which we don't have but I can provide questions/answers instead)

thanks for the feedback everyone! I'll do my best to re-write accordingly

**Comment by jakewhelan** [comment]
Heres the code, it’s open source - you’re welcome to use it 😄

https://github.com/jakewhelan/sfi-cookie-clicker

**Comment by dmateusp** [comment]
made big structure / content changes taking in account the feedback

**Comment by tmccartan** [comment]
LGTM 👍

**Comment by KarlSimon100** [comment]
I like it!

**Pull request merged**